gtfu/src/main/java/gtfu/Util.java

```diff
@@ -846,4 +847,14 @@ public static String getGTFSURL(AgencyData[] list, String id) {
         return collections;
     }
 
-}
\ No newline at end of file
+    public static Map<String, Object> parseJSONasMap(String path) {
+        try {
+            TypeReference<HashMap<String, String[]>> typeRef
+              = new TypeReference<HashMap<String, String[]>>() {};
```

Comment:
This would mean we're limiting ourselves to parsing JSON objects that contain lists of strings as their values, but we want to be able to parse all kinds of objects, including nested ones.

Reply:
This bit turned out to be the only tough part of the edits: getting the object converted to an array of strings without making things too ugly. This is the cleanest I could get it, but I can go back to the drawing board based on this feedback.

Reply:
It seems much more elegant to be able to parse arbitrary JSON objects into a map. I would assume the array of recipients in the JSON is parsed into a java Collection object. If that's the case, you should be able to use `Collection.toArray()`. If for some reason that's not possible then add a little utility function to `Recipients` that does the conversion.

Reply:
Please do the right thing and then merge. Unless you want to have this PR sitting until Jan 3...